securing/part2.html

```diff
@@ -788,7 +788,10 @@ <h1 class="panel-title">
     Account to = accountRepository.findOne(toAccount);
 
     from.setBalance(from.getBalance() - amount);
-    to.setBalance(to.getBalance() - amount);
+    to.setBalance(to.getBalance() + amount);
+    
+    accountRepository.save(from);
```

Comment:
We don't need add these saves here because the `@Transactional` annotation does this automatically.